CSS/CSS.sublime-syntax

```diff
@@ -125,7 +125,7 @@ variables:
 
   # Foreign Tags
   # https://html.spec.whatwg.org/multipage/parsing.html#tag-name-state
-  tag_name_begin: (?=[[:alpha:]])
+  tag_name_begin: (?=[A-Za-z])
```

Comment:
I'm pretty sure HTML allows unicode tag names, no?

Reply:
These patterns were originally copied from _HTML (Plain).sublime-syntax_, which implements tag parsing according to https://html.spec.whatwg.org/multipage/parsing.html#tag-open-state.

Those rules start tag names only with ASCII letters. Any following char can be unicode.